import math
def calculate_complementary_error_function(x):
        return math.erfc(x)
numbers = [1, 2, 3, 4, 5]
print("Max:", max(numbers))